### PR TITLE
first (added hycord)

### PR DESCRIPTION
**Pull request by DeDiamondPro**

### CODE REVIEW

**Comment by isXander** [comment]
please use updated schema

**Comment by DeDiamondPro** [comment]
> please use updated schema

Don't blame me it updated after I created this smh

File: assets/mods.json [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+    "platforms": {
+      "forge": {
+        "1.8.9": {
+          "1.6": "https://static.sk1er.club/repo/mods/patcher/1.6/1.8.9/Patcher-1.6%20(1.8.9).jar"
+        }
+      }
+    }
+  }
+},
+  "hycord": {
```
Comment (isXander) [inline assets/mods.json]: smh missing colon

Comment (DeDiamondPro) [reply assets/mods.json]: I AM CONFUSE I NO SEE PROBLEM

**Pull request merged**